doc/source/dev/toolchain.rst

```diff
@@ -57,6 +57,7 @@ available in release 1.2.x, which is a long-term support release [1]_, [2]_.
  2019              Py3.5+ (but Py2.7-specific code not removed)
  2020              Py3.6+ (removal of Py2.7-specific code permitted)
  2021              Py3.7+
+ 2022              Py3.8+
```

Comment:
This anticipates Python 3.7 being dropped at the end of 2021 as per [NEP29](https://numpy.org/neps/nep-0029-deprecation_policy.html#support-table)

Reply:
Yes, `3.7` has already been removed from `master` of the wheels repo.